### ISSUE TITLE
Council change Elio ↔️ Alex

### ISSUE THREAD

**Issue by ioana-chiorean** [mozilla/Reps#289]
Replacing a Council Member

Steps 4-7 from SOP applies:
- [x] Module owner must select a former Council member to fill the available seat until the next election
- [x] Module owner updates the portal and wiki
- [x] Module owner must inform reps-mentors that the list of Council members has been updated
- [x] Module owner informs reps-general that the list of Council Members has been updated

**Comment by ioana-chiorean** [comment]
1) Discussed with peers and made a list of previous council members. Laka accepted the responsibility. 
2) Wiki https://wiki.mozilla.org/ReMo/Council/History#Eleventh_Council_-_.28October_2017.29
3) Informed all reps ( includes mentors) 
4) Informed all reps

> 
> TL;DR
> 
>     Elio stepped down from Council due to personal reasons
>     Alex will fill this seat until the next election
> 
> Long Version
> Please see https://discourse.mozilla.org/t/important-council-change-elio-alex/25782
>
